### PR TITLE
Config Changes

### PR DESCRIPTION
**Pull request by msusman1**
Update homeserver URI, package name, app name, logo, and color scheme

### CODE REVIEW

File: app/build.gradle.kts [hunk 1]
```diff
@@ -40,13 +40,13 @@ plugins {
 setupKover()
 
 android {
-    namespace = "io.element.android.x"
+    namespace = "org.therevivalists.app"
 
     defaultConfig {
         applicationId = if (isEnterpriseBuild) {
-            "io.element.enterprise"
+            "org.therevivalists.app.enterprise"
```
Comment (iamtalhaasghar) [inline app/build.gradle.kts:47]: kindly remove `.enterprise` @msusman1

File: app/build.gradle.kts [hunk 2]
```diff
@@ -98,9 +98,9 @@ android {
     }
 
     val baseAppName = if (isEnterpriseBuild) {
-        "Element Enterprise"
+        "The Revivalists Enterprise"
```
Comment (iamtalhaasghar) [inline app/build.gradle.kts:101]: Kindly remove `Enterprise`

**Comment by msusman1** [comment]
This is only for Enterprise Build, do we have to remove this logic?

File: features/verifysession/impl/src/main/kotlin/io/element/android/features/verifysession/impl/VerifySelfSessionPresenter.kt [hunk 3]
```diff
@@ -117,7 +117,7 @@ class VerifySelfSessionPresenter @AssistedInject constructor(
         return VerifySelfSessionState(
             verificationFlowStep = verificationFlowStep,
             signOutAction = signOutAction.value,
-            displaySkipButton = buildMeta.isDebuggable,
+            displaySkipButton = true,
```
Comment (iamtalhaasghar) [inline features/verifysession/impl/src/main/kotlin/io/element/android/features/verifysession/impl/VerifySelfSessionPresenter.kt:120]: what this button is for?

Comment (msusman1) [reply features/verifysession/impl/src/main/kotlin/io/element/android/features/verifysession/impl/VerifySelfSessionPresenter.kt:120]: to disable identity verification

File: app/src/main/kotlin/io/element/android/x/initializer/TracingInitializer.kt [hunk 4]
```diff
@@ -19,7 +19,7 @@ import io.element.android.libraries.core.meta.BuildType
 import io.element.android.libraries.matrix.api.tracing.TracingConfiguration
 import io.element.android.libraries.matrix.api.tracing.TracingFilterConfigurations
 import io.element.android.libraries.matrix.api.tracing.WriteToFilesConfiguration
-import io.element.android.x.BuildConfig
+import org.therevivalists.app.BuildConfig
```
Comment (iamtalhaasghar) [inline app/src/main/kotlin/io/element/android/x/initializer/TracingInitializer.kt:22]: can't we keep the original package name in dirs and just change the apk's package name?

Comment (msusman1) [reply app/src/main/kotlin/io/element/android/x/initializer/TracingInitializer.kt:22]: BuildConfig is generated on the fly based on the package name

File: appconfig/src/main/kotlin/io/element/android/appconfig/ApplicationConfig.kt [hunk 5]
```diff
@@ -16,19 +16,19 @@ object ApplicationConfig {
      * - "Element X dbg" for debug builds;
      * - "Element X nightly" for nightly builds.
      */
-    const val APPLICATION_NAME: String = ""
+    const val APPLICATION_NAME: String = "The Revivalists"
```
Comment (iamtalhaasghar) [inline appconfig/src/main/kotlin/io/element/android/appconfig/ApplicationConfig.kt:19]: lets keep this empty and change `R.string.app_name` instead

File: libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/atomic/atoms/ElementLogoAtom.kt [hunk 6]
```diff
@@ -83,21 +83,8 @@ fun ElementLogoAtom(
         )
         Image(
             modifier = Modifier
-                .size(size.logoSize)
```
Comment (iamtalhaasghar) [inline libraries/designsystem/src/main/kotlin/io/element/android/libraries/designsystem/atomic/atoms/ElementLogoAtom.kt:86]: this is a weird shape that appears behind logo on onboarding screen. Good riddance!

**Comment by iamtalhaasghar** [comment]
for now we can ignore this but this might cause issues in future https://github.com/revivalistsorg/element-x-android/issues/7

**Pull request merged**